notebooks/2_feature_engineering.ipynb

```diff
@@ -140,34 +148,40 @@
     "* Unstructured data should first be featurized before consumption in recommenders.  \n",
     "* For text data, you can leverage [TextWiser](https://github.com/fidelity/textwiser) to create text embeddings of item representations.\n",
     "* TextWiser ([AAAI'21](https://ojs.aaai.org/index.php/AAAI/article/view/17814)) provides an easy-to-use API for a rich set of text featurization methods and their transformation while taking advantage of state-of-the-art pretrained NLP models.\n",
-    "* Let's explore a quick start example. "
-   ]
+    "* Let's install the library and explore a quick start example."
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   }
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "# Install TextWiser\n",
+    "!pip install -q textwiser;"
+   ],
    "metadata": {
+    "collapsed": false,
     "pycharm": {
      "name": "#%%\n"
     }
-   },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "['Some document', 'More documents. Including multi-sentence documents.']\n",
-      "[[-0.198138    0.25826374 -0.16164406 ... -0.18344353 -0.04831419\n",
-      "   0.27247027]\n",
-      " [-0.38283753 -0.03924167 -0.10620049 ... -0.25401607  0.21510457\n",
-      "   0.4452556 ]]\n"
-     ]
-    }
-   ],
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
    "source": [
     "# Conceptually, TextWiser is composed of an Embedding, potentially with a pretrained model,\n",
     "# that can be chained into zero or more Transformations\n",
     "from textwiser import TextWiser, Embedding, Transformation, WordOptions, PoolOptions\n",
+    "from transformers import logging\n",
```

Comment:
I dropped a comment on TextWiser side, not super critical, but it seems if these transformers business can be removed from here if TextWiser handles this. I think that would be nice, because it is not clear what to do with transformer lib here while the section is about TWiser.

Reply:
@takojunior  @dorukkilitcioglu

Reply:
Caught up with @takojunior, we can remove these lines as there's a TW branch that will handle this.

Reply:
Cool thank you both! Glad to play a part in making the connection ;)